packages/data-connect/test/runTest.mjs

```diff
@@ -0,0 +1,88 @@
+import { uuidv4 } from '@firebase/util';
+import fetch from 'node-fetch';
```

Comment:
FYI we're removing node-fetch as a dependency in v11. Can you use native fetch here? This would be breaking if it was in user-facing code but I think this is just a test, and our CI is already using Node 18+ so it should be safe to use native fetch.

Reply:
Deleted file.